layouts/partials/comment/waline.html

```diff
@@ -0,0 +1,16 @@
+<div id="waline-comments" class="mt-4"></div>
+{{- $assets := .Site.Data.assets }}
+<script defer src="{{ printf $assets.waline.js.url $assets.waline.version }}" {{ with $assets.waline.js.sri }}
+  integrity="{{ . }}" {{ end }} crossorigin></script>
+{{- with .Site.Params.comment.waline }}
+<script>
+  document.addEventListener("DOMContentLoaded", function () {
+    new Waline({
+      el: "#waline-comments",
+      requiredMeta: [ 'nick','mail' ],
+      dark: "html[class=\"fontawesome-i2svg-active fontawesome-i2svg-complete dark\"]",
```

Comment:
Do we need the `fontawesome-i2svg-active fontawesome-i2svg-complete` here? Eureka only uses `dark` to indicate that it is dark mode.

And maybe we need to make it configurable and add a new possible value (maybe `eureka`) to do this instead of doing it by default. You can refer to how utterances in eureka handles this.

Reply:
It's really not elegant enough here, I think it would be better to change it to `dark: "html.dark"`.

Reply:
And make it configurable is a good idea, I think move `dark: "html.dark"` to `params.yaml` maybe more concise. Just leave it blank when don't need it.

Reply:
And you also need to handle the possible duplication of this param since you manually added it and users may set it in config.